client/app/competitions/[id]/events/page.tsx

```diff
@@ -32,28 +32,35 @@ const ContestEventsPage = async ({ params }: { params: { id: string } }) => {
             {contest.events.map((compEvent) =>
               compEvent.rounds.map((round, roundIndex) => {
                 const cutoffText = round.cutoff
-                  ? `${round.cutoff.numberOfAttempts} ${
-                    round.cutoff.numberOfAttempts === 1 ? 'attempt' : 'attempts'
+                  ? `${round.cutoff.numberOfAttempts} ${round.cutoff.numberOfAttempts === 1 ? 'attempt' : 'attempts'
```

Comment:
Are you using Prettier and is it reading the prettierrc file in the root of the project? This change seems unintentional.

Reply:
Same for some of the other changes in this PR

Reply:
I ran a format via VSCode, and then committing ran ESLint but not sure about Prettier

Reply:
Would you mind installing the Prettier vs code plugin, setting it as your default formatter and reformatting this file?